test/test_common/environment.cc

```diff
@@ -49,6 +48,30 @@ void TestEnvironment::initializeOptions(int argc, char** argv) {
   argv_ = argv;
 }
 
+bool TestEnvironment::isTestIpVersionOnly(const Network::Address::IpVersion& type) {
+  const char* option = ::getenv("IP_TEST_TYPE");
```

Comment:
`getCheckedEnvVar` will both do the `::getenv`, check it's not null and convert to `std::string`.